websites/L/LiveLinkBio/dist/metadata.json

```diff
@@ -0,0 +1,36 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "Dark_Ville",
+		"id": "638080361179512853"
+	},
+	"service": "LiveLinkBio",
+	"description": {
+		"en": "livelinkbio is an advanced URL Shortener Platform, that allows you to easily shorten links, target your audience based on their location or platform, and provide analytics insights for the shortened links.",
+		"nl": "livelinkbio is een geavanceerde URL Verkorter Platform, waar je makkelijk links kan verkorten, om jouw gekozen publiek te bereiken op basis van locatie en platform en het bied analytics aan voor de verkorte links."
```

Comment:
@Bas950

Reply:
The Dutch translation should ideally be `LiveLinkBio is een geavanceerd URL-verkortingsplatform waar je makkelijk links kan verkorten. Bereik je doelgroep op basis van locatie of platform en ontvang inzichten voor je verkorte links.`